code-orange/VisitedLocationPanel.swift

```diff
@@ -0,0 +1,233 @@
+    button.addTarget(self, action: #selector(nextLocationPickedTapped), for: .touchUpInside)
+    return button
+  }()
+
+  private lazy var pagesLabel: UILabel = {
+    let label = UILabel()
+    label.font = UIFont.systemFont(ofSize: 16)
+    return label
+  }()
+
+  private lazy var addressLabel: UILabel = {
+    let label = UILabel()
+    label.font = UIFont.boldSystemFont(ofSize: 16)
+    return label
+  }()
+  
+  private lazy var timeFormatter: DateFormatter = {
+    let dateFormater = DateFormatter()
+    dateFormater.dateFormat = "HH:MM"
+    return dateFormater
+  }()
+
+  private lazy var timeframeLabel: UILabel = {
+    let label = UILabel()
+    label.font = UIFont.systemFont(ofSize: 14)
+    return label
+  }()
+
+  private lazy var topLabelStack: UIStackView = {
+    let paragraphStyle = NSMutableParagraphStyle()
+    paragraphStyle.alignment = .justified
+    paragraphStyle.baseWritingDirection = .rightToLeft
+    let attributedText = NSMutableAttributedString(string: "ייתכן שנחשפת באזור לוירוס הקורונה.\n", attributes: [NSAttributedString.Key.font: UIFont.boldSystemFont(ofSize: 16), NSAttributedString.Key.paragraphStyle: paragraphStyle])
+    attributedText.append(NSAttributedString(string: "שים לב, ההודעה אינה אומרת שנדבקת.", attributes: [NSAttributedString.Key.font: UIFont.systemFont(ofSize: 16), NSAttributedString.Key.paragraphStyle: paragraphStyle]))
+    let label = UILabel()
+    label.attributedText = attributedText
+    label.numberOfLines = 0
+    return HorizontalPaddingStackView(around: label)
+  }()
+
+  private lazy var bottomLabelStack: UIStackView = {
+    let paragraphStyle = NSMutableParagraphStyle()
+    paragraphStyle.alignment = .justified
+    paragraphStyle.baseWritingDirection = .rightToLeft
+    let attributedText = NSMutableAttributedString(string: "צעדים שכדאי לעשות כרגע:\n", attributes: [NSAttributedString.Key.font: UIFont.boldSystemFont(ofSize: 14), NSAttributedString.Key.paragraphStyle: paragraphStyle])
+    attributedText.append(NSAttributedString(string: "להיכנס לבידוד, ולמדוד את חום הגוף. אם זיהית תסמינים (חום, שיעול יבש) יש לפנות למד״א.", attributes: [NSAttributedString.Key.font: UIFont.systemFont(ofSize: 14), NSAttributedString.Key.paragraphStyle: paragraphStyle]))
+    let label = UILabel()
+    label.attributedText = attributedText
+    label.numberOfLines = 0
+    return HorizontalPaddingStackView(around: label)
+  }()
+
+  private lazy var healthAdministrationWebsiteButton: UIButton = {
+    let button = UIButton()
+    button.backgroundColor = .nxOrange
+    button.setTitleColor(.black, for: .normal)
+    button.setTitle("אתר משרד הבריאות", for: .normal)
+    button.titleLabel?.font = UIFont.systemFont(ofSize: 14)
+    button.addTarget(self, action: #selector(healthAdministrationWebsiteButtonTapped), for: .touchUpInside)
+    return button
+  }()
+
+  private lazy var callEmergencyServiceButton: UIButton = {
+    let button = UIButton()
+    button.backgroundColor = .nxOrange
```

Comment:
change to orange?

Reply:
nope - different colors. Design is nxOrange which is much brighter